serenityff/charge/gnn/utils/rdkit_helper.py

```diff
@@ -21,6 +22,69 @@ def mols_from_sdf(sdf_file: str, removeHs: Optional[bool] = False) -> Sequence[M
     return Chem.SDMolSupplier(sdf_file, removeHs=removeHs)
 
 
+def get_mol_prop_as_array(prop_name: Optional[str], mol: Chem.Mol, dtype: type = float) -> np.ndarray:
+    """Get atomic properties from an RDKit molecule object as an array.
+
+    The property is expected to be a string of '|' separated numerical
+    values, one for each atom in the molecule.
+
+    Parameters
+    ----------
```

Comment:
You are not consistent with the documentation style. I don't care which one is used in DASH, but it would be nice if it's only one and not everyone. 

On second thought this repo is already a wild mix 🙈